src/devices/lacrosse.c

```diff
@@ -180,10 +180,10 @@ static int lacrossetx_callback(bitbuffer_t *bitbuffer) {
 			// message integrity.
 			if (msg_nybbles[5] != msg_nybbles[8] || 
 			    msg_nybbles[6] != msg_nybbles[9]) {
-				if (debug_output) {
-			    fprintf(stderr,
-				    "LaCrosse TX Sensor %02x, type: %d: message value mismatch int(%3.1f) != %d?\n",
-				    sensor_id, msg_type, msg_value, msg_value_int);
+				if (debug_output >= 1) {
```

Comment:
No.  shouldn't require -D -D,   you want to be able to see cases like this (where there are message types you haven't seen before), without higher levels of debugging.   Again, not overly verbose.